persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_65/main.bal

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2022 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Fixed in [01fd48a](https://github.com/ballerina-platform/persist-tools/pull/313/commits/01fd48ae29ea561c5cc0ceb5f600cc0e8072bf45)